p2panda-encryption/src/two_party/x3dh.rs

```diff
@@ -34,6 +34,9 @@ pub struct X3dhCiphertext {
     /// in long-term key bundles).
     pub onetime_prekey_id: Option<OneTimePreKeyId>,
 
+    /// Identifier of the used long-term pre-key.
+    pub prekey_id: PreKeyId,
```

Comment:
This satisfies partly point 2: We know now _which_ pre-key was used during X3DH key agreement